cql2pgjson/src/test/java/org/folio/cql2pgjson/CompoundIndexTest.java

```diff
@@ -29,7 +56,10 @@ public void SQLExpressionGIN() throws Exception {
     CQL2PgJSON cql2pgJson = new CQL2PgJSON("tableb");
     cql2pgJson.setDbSchemaPath("templates/db_scripts/compoundIndex.json");
     String sql = cql2pgJson.toSql("address == \"Boston MA\"").toString();
-    String expected = "WHERE lower(concat_space_sql(jsonb->>'city', jsonb->>'state')) LIKE lower('Boston MA')";
+    String expected = "WHERE CASE WHEN length(lower('Boston MA')) <= 600 THEN left(lower(concat_space_sql(jsonb->>'city', jsonb->>'state')),600) "
+        + "LIKE lower('Boston MA') ELSE left(lower(concat_space_sql(jsonb->>'city', jsonb->>'state')),600)  "
+        + "LIKE  left(lower('Boston MA'),600) AND lower(concat_space_sql(jsonb->>'city', jsonb->>'state'))  "
+        + "LIKE  lower('Boston MA') END";
     assertEquals(expected, sql);
```

Comment:
This unit test is about an sqlExpession. The CASE WHEN ELSE END with left(...) has already been tested in multiFieldNames* method above and should therefore not pollute this unit tests.
Reducing the test to the relevant part makes it more easy to understand and maintain. Example:
    
    String expected = "lower(concat_space_sql(jsonb->>'city', jsonb->>'state')) LIKE lower('Boston MA')";
    assertThat(sql, containsString(expected));